stream_alert_cli/terraform/athena.py

```diff
@@ -13,9 +13,7 @@
 See the License for the specific language governing permissions and
 limitations under the License.
 """
-from stream_alert.shared import metrics
-from stream_alert_cli.manage_lambda.package import AthenaPackage
-from stream_alert_cli.terraform.common import DEFAULT_SNS_MONITORING_TOPIC, infinitedict
+from stream_alert_cli.terraform.common import infinitedict
 
 
 def generate_athena(config):
```

Comment:
the dictionary created in generate_athena on line 28 is an infinitedict, and it takes advantage of the format mostly so that the nested access can be done on the fly rather than creating the nests